packages/tables/resources/views/components/filters/index.blade.php

```diff
@@ -2,7 +2,7 @@
     'form'
 ])
 
-<div class="space-y-6">
+<div class="space-y-6 filament-tables-filters">
```

Comment:
```suggestion
<div class="space-y-6 filament-tables-filters-form">
```

`filament-tables-filters` is already used for the main popover